Bug Fix: Minified JSX Error on Netifly Build

When built and hosted on Netlify, it shows a blank white screen.

Refer to here for possible error solution:
Minified React error #152 - comments before JSX #8687
https://github.com/facebook/create-react-app/issues/8687

This is console log error output:
```
react-dom.production.min.js:209 Error: Minified React error #152; visit https://reactjs.org/docs/error-decoder.html?invariant=152&args[]=a for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at react-dom.production.min.js:149
    at Oo (react-dom.production.min.js:173)
    at Uo (react-dom.production.min.js:182)
    at Bo (react-dom.production.min.js:181)
    at gs (react-dom.production.min.js:263)
    at lu (react-dom.production.min.js:246)
    at su (react-dom.production.min.js:246)
    at Js (react-dom.production.min.js:239)
    at react-dom.production.min.js:123
    at t.unstable_runWithPriority (scheduler.production.min.js:19)
es @ react-dom.production.min.js:209
react-dom.production.min.js:123 Uncaught Error: Minified React error #152; visit https://reactjs.org/docs/error-decoder.html?invariant=152&args[]=a for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at react-dom.production.min.js:149
    at Oo (react-dom.production.min.js:173)
    at Uo (react-dom.production.min.js:182)
    at Bo (react-dom.production.min.js:181)
    at gs (react-dom.production.min.js:263)
    at lu (react-dom.production.min.js:246)
    at su (react-dom.production.min.js:246)
    at Js (react-dom.production.min.js:239)
    at react-dom.production.min.js:123
    at t.unstable_runWithPriority (scheduler.production.min.js:19)
```

--- a/src/Components/DraggablePlaylist.js
+++ b/src/Components/DraggablePlaylist.js
@@ -312,14 +312,6 @@ removeListEntry = (sPlaylist, pPlaylist) =>{
     idx+=1
   }
   return null
-  // var pVideoIds = pPlaylist.map((video, idx)=>{ return video.videoId })
-  // Object.fromEntries(
-  //   Object.entries(obj).map(
-  //     ([k, v], i) => [k, v => 2 * v  ]
-  //   )
-  // )
-  // console.log(pVideoIds )
-  // return pPlaylist
 }
 
   

--- a/src/Containers/DraggablePlaylistContainer.js
+++ b/src/Containers/DraggablePlaylistContainer.js
@@ -47,8 +47,7 @@ export default class DraggablePlaylistContainer extends React.Component {
 
     console.log("player3: ", this.player)
     
-    return (
-    //   <div className={`playlistContainerMaster ${!this.state.open ? ' closed ' : ' open '}`}>	
+    return (	
       <div className={`playlistContainerMaster ${!this.state.open ? ' open ':' closed '}`}>	
         <div className="playlistBar" onClick={this.togglePlaylistVisibility} > Playlist </div>
         

